tests/test_restaurants.py

```diff
@@ -10,7 +10,7 @@ def test_post(test_client):
     response = test_client.post('api/restaurants', json=data)
     last_element = len(response.get_json()) - 1
     assert response.status_code == 201
-    assert all([value == response.get_json()[last_element][key] for key, value in data.items()])
+    assert all([value == response.get_json()[key] for key, value in data.items()])
```

Comment:
can be just `response.get_json() == data`

Reply:
data doesn't have id